test/rtu-client-request-handler.test.js

```diff
@@ -81,8 +82,9 @@ describe('Modbus/RTU Client Request Tests', function () {
 
           done()
         }).catch(function (err) {
-          // Exception type should be ModbusException not crcMismatch or any other 
+          // Exception type should be ModbusException not crcMismatch or any other
           assert.equal(err.err, 'ModbusException')
+          assert.equal(err.request instanceof Modbus.ModbusRTURequest, true)
```

Comment:
I've added asserts for testing if the actual object is the same.

I also added 2 additional tests for TCP where this is tested. I had to write these tests differently than the other tests since I can't get access to the actual request body when using something like `tcpClient.readCoils(..)`